src/amo/components/DefinitionList/styles.scss

```diff
@@ -5,10 +5,15 @@
   padding: 0;
 
   ul {
+    break-inside: avoid;
```

Comment:
mmm?  We're change `<ul>` too?

Reply:
https://github.com/mozilla/addons-frontend/pull/13705/commits/ed346565f93b3fd170d03fe8fe7cc1e070698105